dbt_semantic_interfaces/protocols/metric.py

```diff
@@ -117,6 +117,23 @@ def post_aggregation_reference(self) -> MetricReference:
         pass
 
 
+class ConstantPropertyInput(Protocol):
+    """Provides the constant property values for conversion metrics.
+
+    The specified property will be a reference of a dimension/entity.
```

Comment:
I don't think anybody other than us will know what a `constant property` is without a more explicit description. Maybe something like:

```
"""
Provides the constant property set for conversion metrics.

Constant properties are additional elements linking a base event to a conversion event. 
The specified properties will typically be a reference to a dimension or entity, and will be used
to join the base event to the final conversion event. Typical constant properties are things like
session keys (for services where conversions are measured within a user session), or secondary entities
(like a user/application pair for an app platform or a user/shop pair for a retail/online storefront platform).
"""
```

Reply:
I would not have known what they were without this context ➕